config/routes.rb

```diff
@@ -2,7 +2,7 @@
   root 'homes#index'
   devise_for :users
 
-  resources :bucket_lists, only: [:index, :new, :create, :edit]
+  resources :bucket_lists, only: [:index, :new, :create, :edit, :delete, :destroy]
 
-  patch '/bucket_lists/:id/edit', to: 'bucket_lists#update', as: 'bucket_list'
+  patch '/bucket_lists/:id/edit', to: 'bucket_lists#update'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.